definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "30/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "SOTAgreementPayAndSubmitRequired",
```

Comment:
@madjava was this not required by Sateesh as a must for SaC?

Reply:
CAA users are not required whatsoever for Share a Case - they are only required for RJ

Reply:
I'm removing those anyway as part of https://github.com/hmcts/div-ccd-definitions/pull/635/files even for resp journey because those specific states come before "AOSAwaiting" so respondent solicitor should not see them

Reply:
Can you list which states these should be and I'll remove them as part of this PR. This PR will most likely be merged before PR-635 as 635 is blocked until we can login as a respondent solicitor user which I can't see happening until early next week at least